.buildbot.sh

```diff
@@ -24,18 +24,19 @@ tar -xf llvm-project-13.0.0.src.tar.xz
 cd llvm-project-13.0.0.src
 mkdir -p build
 cd build
-cmake -DCMAKE_BUILD_PARALLEL_LEVEL=24 -DLLVM_ENABLE_PROJECTS="clang;llvm;lld;clang-tools-extra" -DLLVM_LINK_LLVM_DYLIB=ON -G "Unix Makefiles" ../llvm/.
-cmake --build .
+cmake -DLLVM_ENABLE_PROJECTS="clang;llvm;lld;clang-tools-extra" -DLLVM_LINK_LLVM_DYLIB=ON -G "Unix Makefiles" ../llvm/.
+cmake --build . --parallel 24
```

Comment:
Let's not hard-code core counts in: `nproc` is a nice way of saying "use all cores". But, I think, `cmake` might just parallelise things semi-appropriately anyway?

Reply:
It's significantly slower without `--parallel`.

Reply:
Is there a way of saying "cmake please use all cores"?

Reply:
I didn't want to use all the cores (32) otherwise we can't do anything else on `bencher14` while testing. I'll check.